src/python/pants/pantsd/pants_daemon.py

```diff
@@ -354,16 +358,15 @@ def _pantsd_logging(self) -> Iterator[IO[str]]:
             # is launched with full_init=True. If PantsdDaemon is launched with
             # full_init=True, we can guarantee self._native is non-None.
             native = cast(Native, self._native)
-            log_handler = setup_logging(
-                self._log_level,
-                native=native,
-                log_dir=self._log_dir,
-                log_filename=self.LOG_NAME,
-                warnings_filter_regexes=self._bootstrap_options.for_global_scope(),  # type: ignore[union-attr]
-            )
-            # We know log_handler is never None because we did pass a non-None `log_dir`
-            # to setup_logging.
-            log_handler = cast(NativeHandler, log_handler)
+
+            level = self._log_level
+            # TODO(gregs) - what is the right value for this warnings_filter_regexes line?
+            # warnings_filter_regexes=self._bootstrap_options.for_global_scope(),  # type: ignore[union-attr]
```

Comment:
This is commented out. I don't think you intended for it to be.

Reply:
`warnings_filter_regexes=self._bootstrap_options.for_global_scope()` is in the current logging setup code in `pants_daemon.py`, which doesn't make any sense to me and doesn't typecheck. I think this should be `warnings_filter_regexes=self._bootstrap_options.for_global_scope().ignore_pants_warnings` , but I wanted to get @stuhood 's eyes on this in case there's something I'm missing here.

Reply:
Ah. Yes, it should be.